moveit_ros/hybrid_planning/package.xml

```diff
@@ -0,0 +1,48 @@
+<package format="2">
+  <name>moveit_hybrid_planning</name>
+  <version>0.0.1</version>
```

Comment:
Please set this to the version used in the rest of the repo (all packages have the same version).